Packs/Endace/Playbooks/playbook_Endace_Search_Archive_Download.yml

```diff
@@ -0,0 +1,606 @@
+    view: "{\n  \"position\": {\n    \"x\": 275,\n    \"y\": 1595\n  }\n}"
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  '35':
+    id: '35'
+    taskid: 1753b4b1-95f3-40ee-86b3-f1deee0109bf
+    type: regular
+    task:
+      id: 1753b4b1-95f3-40ee-86b3-f1deee0109bf
+      version: -1
+      name: endace-download-pcap
+      description: Download a copy of the PCAP file from EndaceProbe if PCAP file
+        size is within the threshold value defined by filesizelimit.
+      script: Endace|||endace-download-pcap
+      type: regular
+      iscommand: true
+      brand: Endace
+    nexttasks:
+      '#none#':
+      - '25'
+    scriptarguments:
+      filename:
+        simple: ${Endace.Archive.Task.FileName}
+      filesizelimit: {}
+    separatecontext: false
+    view: "{\n  \"position\": {\n    \"x\": 655,\n    \"y\": 2120\n  }\n}"
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  '36':
+    id: '36'
+    taskid: ae66399d-5356-4576-81d4-bababf3f564d
+    type: regular
+    task:
+      id: ae66399d-5356-4576-81d4-bababf3f564d
+      version: -1
+      name: endace-create-archive
+      description: Create an archive task to archive packets of interest on EndaceProbe.
+        Archived packets can later be downloaded from EndaceProbe as a PCAP file.
+        Archived Files never expire. Allowed chars are text, numbers, dash and underscore.
+      script: Endace|||endace-create-archive
+      type: regular
+      iscommand: true
+      brand: Endace
+    nexttasks:
+      '#none#':
+      - '24'
+    scriptarguments:
+      archive_filename:
+        simple: pcaparchive
+      dest_host_list:
+        simple: ${inputs.dest_host_list}
+      dest_port_list:
+        simple: ${inputs.dest_port_list}
+      end:
+        simple: ${inputs.end}
+      protocol:
+        simple: ${inputs.protocol}
+      src_host_list:
+        simple: ${inputs.src_host_list}
+      src_port_list:
+        simple: ${inputs.src_port_list}
+      start:
+        simple: ${inputs.start}
+      timeframe:
+        simple: ${inputs.timeframe}
+    separatecontext: false
+    view: "{\n  \"position\": {\n    \"x\": 275,\n    \"y\": 1070\n  }\n}"
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+view: "{\n  \"linkLabelsPosition\": {\n    \"14_7_no\": 0.32,\n    \"26_7_no\": 0.1,\n\
+  \    \"27_7_no\": 0.25,\n    \"9_26_yes\": 0.67,\n    \"9_7_no\": 0.1\n  },\n  \"\
+  paper\": {\n    \"dimensions\": {\n      \"height\": 2485,\n      \"width\": 985,\n\
+  \      \"x\": 50,\n      \"y\": 50\n    }\n  }\n}"
+inputs:
+- key: start
+  value: {}
+  required: false
+  description: Event Start Time in ISO 8601 format
+  playbookInputQuery:
+- key: end
+  value: {}
+  required: false
+  description: Event End Time in ISO 8601 format
+  playbookInputQuery:
+- key: src_host_list
```

Comment:
Have you considered adding an argument just for an ip? regardless if its source or destination in case we dont know if source or destination we will just need to run the playbook twice.

Reply:
We may do it as a feature request.